src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -133,7 +133,7 @@ protected void setPrivateFieldValue(
         ReflectionHelpers.setPrivateFieldValue(cls, this, fieldName, newValue);
     }
 
-    protected Map<String, CommandInfo> getAdditionalCommands() {
+    public Map<String, CommandInfo> getAdditionalCommands() {
```

Comment:
I assume there we can safely delete this getter as it is public now in the parent class

Reply:
yes, we can, but it will bump the minimal Selenium client version to `4.28.0-SNAPSHOT`, if it's ok, I'll do it

Reply:
lets wait until 4.28 is released, so then we could also release a new minor version of java client and set the minimum version there